embedded-memsql/src/main/java/com/playtika/test/memsql/EmbeddedMemSqlBootstrapConfiguration.java

```diff
@@ -60,14 +61,16 @@ MemSqlStatusCheck memSqlStartupCheckStrategy(MemSqlProperties properties) {
     @Bean(name = BEAN_NAME_EMBEDDED_MEMSQL, destroyMethod = "stop")
     public GenericContainer memsql(ConfigurableEnvironment environment,
                                    MemSqlProperties properties,
-                                   MemSqlStatusCheck memSqlStatusCheck) {
+                                   MemSqlStatusCheck memSqlStatusCheck,
+                                   Network network) {
```

Comment:
make it Optional. It may be absent

Reply:
https://github.com/Playtika/testcontainers-spring-boot/blob/develop/embedded-kafka/src/main/java/com/playtika/test/kafka/configuration/KafkaContainerConfiguration.java#L136
here it is not optional other wise containers will be in different networks and will not be available to each other